common/src/models/domain/globals.ts

```diff
@@ -0,0 +1,19 @@
+export class Globals {
+    decodedToken: any;
+    enableAlwaysOnTop: boolean;
+    installedVersion: string;
+    lastActive: number;
+    locale: string;
+    openAtLogin: boolean;
+    organizationInvitation: any;
```

Comment:
It is. Moved to the account model.

Reply:
I had to revert this. `organizationInvite` is set from a url before a login attempt happens. We would need to revisit how we handle invites to make this account centric, which seems out of scope since web won't be getting account switching capabilities.